integration/entrypoint.sh

```diff
@@ -245,13 +264,19 @@ function pull_remove_multiple_images {
     nerdctl --snapshotter nydus image pull "${WORDPRESS_IMAGE}"
     nerdctl --snapshotter nydus image pull "${TOMCAT_IMAGE}"
 
-    nerdctl --snapshotter nydus image rm "${TOMCAT_IMAGE}"
-    nerdctl --snapshotter nydus image rm "${JAVA_IMAGE}"
-    nerdctl --snapshotter nydus image rm "${WORDPRESS_IMAGE}"
+    nerdctl --snapshotter nydus create --rm --net none "${TOMCAT_IMAGE}"
+    nerdctl --snapshotter nydus create --rm --net none "${WORDPRESS_IMAGE}"
 
-    # TODO: Validate running nydusd number
+    nerdctl --snapshotter nydus image rm --force "${JAVA_IMAGE}"
+    nerdctl --snapshotter nydus image rm --force "${WORDPRESS_IMAGE}"
+
+    nerdctl --snapshotter nydus image rm --force --async "${TOMCAT_IMAGE}"
```

Comment:
Can we explain why need `--async`?

Reply:
No purpose to adding `--async` flag here, just want to give a fuzz flag to test if it works well

Reply:
Ok, maybe we can add a comment.